geom/src/bounds.rs

```diff
@@ -231,6 +231,17 @@ impl GPSBounds {
         pts.iter().map(|pt| pt.to_gps(self)).collect()
     }
 
+    pub fn convert_back_xy(&self, x: f64, y: f64) -> LonLat {
+        let (width, height) = {
```

Comment:
What if we just did `Pt2D::new(x, y).to_gps(self)` here? Any reason to copy the math?

Reply:
Edit: you moved the logic here, this is fine!